README.md

```diff
@@ -111,6 +111,7 @@ Alternatively, you can also use [Podman](https://github.com/containers/libpod)
 Now to run a new Fineract instance you can simply:
 
 1. `git clone https://github.com/apache/fineract.git ; cd fineract`
+1. for windows, use `git clone https://github.com/apache/fineract.git --config core.autocrlf=input ; cd fineract`
```

Comment:
@apurv-1 thank you for your first contribution to this project! I'll merge this as-is. 

If you would like to make a 2nd contribution, you could change it to be part of the 1. instead of a 2. step.